Ginger/GingerCoreCommon/VariablesLib/VariableSelectionList.cs

```diff
@@ -41,12 +41,42 @@ public override string VariableType
             get { return "Selection List"; }
         }
 
+        [IsSerializedForLocalRepository(true)]        
+        private bool mIsLoopEnabled = true;
         [IsSerializedForLocalRepository(true)]
-        public bool IsLoopEnabled { get; set; } = true;
+        public bool IsLoopEnabled
+        {
+            get { return mIsLoopEnabled; }
+            set
+            {
+                if (mIsLoopEnabled != value)
+                {
+                    mIsLoopEnabled = value;
+                    OnPropertyChanged(nameof(IsLoopEnabled));
+                }
+            }
+        }
+
+
+
+
 
         [IsSerializedForLocalRepository(false)]
-        public bool IsDynamicValueModificationEnabled { get; set; } = false;
+        private bool mIsDynamicValueModificationEnabled;
```

Comment:
Please remove [IsSerializedForLocalRepository(true)] for mIsLoopEnabled as we have same attribute for IsDynamicValueModificationEnabled

Reply:
Please remove it from this field as well